pixart.js

```diff
@@ -1 +1,38 @@
 
+var color = document.getElementById("color-field");
+
+document.getElementById("set-color").addEventListener("click", function(evt){
+	evt.preventDefault();
+	document.getElementsByClassName("brush")[0].style.backgroundColor = color.value
+	console.log("Yay, you just typed")
+
+})
+
+    for(var x = 0; x < 8000; x++) {
```

Comment:
Be consistent with your indentation! This should be a the root level of indentation (all the way to the left)